### PR TITLE
Detection rules for MITRE's Carbanak step 9.B

### LINKED ISSUE #9353: Detect 9.B - Credentials from Web Browsers
Generate detection rule using the Carbanak [step 9.B](https://github.com/center-for-threat-informed-defense/adversary_emulation_library/tree/master/carbanak/Emulation_Plan/Scenario_1#9b---credentials-from-web-browsers-t1070004)

- [x]  Test environment setup
- [x]  Collect relevant logs
- [x]  Write appropriate decoders and rules

### Substeps

1. Upload `dumpWebCreds.exe` to `cfo` as `infosMin48.exe`
    ```
    meterpreter > upload /home/<attacker>/carbanak/Resources/step9/dumpWebCreds.exe C:\\Users\\<cfo_user>\\AppData\\Local\\Temp\\infosMin48.exe
    ```
   
2. Execute `infosMin48.exe`
    ```
    meterpreter > execute -f C:\\Users\\<cfo_user>\\AppData\\Local\\Temp\\infosMin48.exe -i -H
    ```

2. Delete dropped files
    ```
    meterpreter > execute -f powershell.exe -i -H -a "-c Remove-Item $env:TEMP\* -Recurse -Force -Erroraction 'silentlycontinue'"
    ```

### Detection criteria

9.B.1: explorer.exe downloads infosMin48.exe from 192.168.0.4
9.B.2: infosMin48.exe calls the VaultEnumerateItems API from vaultcli.dll
9.B.3: powershell.exe deletes files from C:\Users\jsmith\AppData\Local\Temp\

### PR DESCRIPTION
**Pull request by fabamatic**
|Related issue|
|---|
|Closes #9353 |

## Description

Added rules 92150 and 92110

- 92150: Detects load of vaultcli.dll module is loaded by a process, it may be used to dump stored passwords
- 92110: Detects Powershell process being used to delete files or folders. Low level rule

## Configuration options
These rules require Sysmon logs to work

### CODE REVIEW

**Review by 72nomada — DISMISSED** [review]
LGTM

**Review by jcruzlp — APPROVED** [review]
LGTM!

**Review by JavierBejMen — APPROVED** [review]
LGTM

File: ruleset/rules/0800-sysmon_id_1.xml [hunk 1]
```diff
@@ -250,4 +250,15 @@
...
+    <rule id="92110" level="0">
+        <if_group>sysmon_event1</if_group>
+        <field name="win.eventdata.originalFileName" type="pcre2">(?i)PowerShell\.EXE</field>
+        <field name="win.eventdata.commandLine" type="pcre2">(?i)(Remove-Item|ri|rm|rmdir|del|erase|rd)\b</field>
```
Comment (NicoGuini) [inline ruleset/rules/0800-sysmon_id_1.xml:258]: We still having a limitation for powershell scriptblock, right?

**Review by NicoGuini — APPROVED** [review]

File: ruleset/rules/0800-sysmon_id_1.xml (hunk 1 above)
Comment (fabamatic) [reply ruleset/rules/0800-sysmon_id_1.xml:258]: Sorry Nico, didn't see this comment. Scriptblock is usable, can be enabled on client side but it's very expensive in log size

**Pull request merged**